cmd/oci-create-runtime-bundle/oci-create-runtime-bundle.1.md

```diff
@@ -18,17 +18,17 @@ runtime-spec-compatible `dest/config.json`.
 **--help**
   Print usage statement
 
-**--ref**
+**--ref**=""
   The ref pointing to the manifest of the OCI image. This must be present in the "refs" subdirectory of the image. (default "v1.0")
 
-**--rootfs**
+**--rootfs**=""
   A directory representing the root filesystem of the container in the OCI runtime bundle. It is strongly recommended to keep the default value. (default "rootfs")
 
-**--type**
+**--type**=""
   Type of the file to unpack. If unset, oci-create-runtime-bundle will try to auto-detect the type. One of "imageLayout,image"
 
-**--version**
-  Print version information and exit.
+**-v**, **--version**=*true*|*false*
+  Print version information and exit. Default is false.
```

Comment:
I don't think user will use `oci-create-runtime-bundle -v=false`, it's a wrong use case and it will print the help messages.
```
$ oci-create-runtime-bundle -v=false
both src and dest must be provided
Usage:
  oci-create-runtime-bundle [src] [dest] [flags]

Flags:
      --ref string      The ref pointing to the manifest of the OCI image. This must be present in the "refs" subdirectory of the image. (default "v1.0")
      --rootfs string   A directory representing the root filesystem of the container in the OCI runtime bundle.
It is strongly recommended to keep the default value. (default "rootfs")
      --type string     Type of the file to unpack. If unset, oci-create-runtime-bundle will try to auto-detect the type. One of "imageLayout,image"
  -v, --version         Print version information and exit

```

and juse check some other common linux tools like `man fdisk`
```
-V, --version
              Display version information and exit.
```

and also the runtime tools https://github.com/opencontainers/runtime-tools/blob/master/man/oci-runtime-tool.1.md